tests/apple-exporter/test-class-exporter-content.php

```diff
@@ -24,7 +24,7 @@ public function testMinimalContent() {
 	}
 
 	public function testCompleteContent() {
-		$content  = new \Apple_Exporter\Exporter_Content( 3, 'Title', '<p>Example content</p>', 'some intro', 'someurl.com' );
+		$content  = new \Apple_Exporter\Exporter_Content( 3, 'Title', '<p>Example content</p>', null, 'some intro', 'someurl.com' );
```

Comment:
Match new function signature for constructor, which puts slug at the end, and defaults to null so we can omit it:
```suggestion
		$content  = new \Apple_Exporter\Exporter_Content( 3, 'Title', '<p>Example content</p>', 'some intro', 'someurl.com' );
```